lib/Indexer/Adapter/Tolerant/Indexer/ClassDeclarationIndexer.php

```diff
@@ -36,29 +35,9 @@ public function index(Index $index, TextDocument $document, Node $node): void
         $this->indexClassInterfaces($index, $record, $node);
         $this->indexBaseClass($index, $record, $node);
 
-        $this->indexAttributes($record, $node);
-
         $index->write($record);
     }
 
-    public function indexAttributes(ClassRecord $record, ClassDeclaration $node): void
-    {
-        $attributes = $node->attributes ?? [];
-        if (count($attributes) === 0) {
-            return;
-        }
-
-        foreach ($attributes as $attributeGroup) {
-            foreach ($attributeGroup->attributes->children as $attribute) {
-                /** @var Attribute $attribute */
-                if ($attribute->name->getText() === 'Attribute') {
-                    $record->addFlag(ClassRecord::FLAG_ATTRIBUTE);
-                    return;
-                }
-            }
-        }
-    }
-
```

Comment:
@dantleech I think you have deleted something here for indexing Attributes that shouldn't have been deleted?

Reply:
Looks like it :thinking:

Reply:
https://github.com/phpactor/phpactor/pull/2156